Dockerfile

```diff
@@ -453,6 +453,11 @@ RUN rm /etc/pgbackrest.conf && ln -s "${PGBACKREST_CONFIG}" /etc/pgbackrest.conf
 ## to make this work nicely, they need to exist though
 RUN for i in $(seq 0 7); do touch "${PGLOG}/postgresql-$i.log" "${PGLOG}/postgresql-$i.csv"; done
 
+## Patch Patroni code with changes from https://github.com/timescale/patroni/pull/1
+## This is a temporary solution until changes land upstream.
+RUN wget -qO- https://raw.githubusercontent.com/timescale/patroni/v2.2.0-beta.0/patroni/ha.py > /usr/lib/python3/dist-packages/patroni/ha.py && \
+    wget -qO- https://raw.githubusercontent.com/timescale/patroni/v2.2.0-beta.0/patroni/config.py > /usr/lib/python3/dist-packages/patroni/config.py
+
```

Comment:
```suggestion
## Patch Patroni code with changes from https://github.com/timescale/patroni/pull/1
## This is a temporary solution until changes land upstream.
USER root
RUN wget -qO- https://raw.githubusercontent.com/timescale/patroni/v2.2.0-beta.0/patroni/ha.py > /usr/lib/python3/dist-packages/patroni/ha.py && \
    wget -qO- https://raw.githubusercontent.com/timescale/patroni/v2.2.0-beta.0/patroni/config.py > /usr/lib/python3/dist-packages/patroni/config.py
USER root
```

Reply:
Thanks boss. I actually decided to just move this code up a bit, so that it appears within the RUN clause which installs Patroni itself.